tasks/main.yml

```diff
@@ -18,6 +12,7 @@
 - name: Add repository
   apt_repository:
     repo: deb https://nginx.org/packages/mainline/{{ ansible_distribution | lower }} {{ ansible_distribution_release }} nginx
+    update_cache: true
```

Comment:
Doesn't ansible do this automatically when you add a repository?

Reply:
This is the default value

Reply:
Too late